### ISSUE TITLE
Map note icon "X" too scary to click on

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/openstreetmap-website#2882]
(We) users are nervous about clicking on anything with an "X" in the fear that it clicking on it will "destroy" it (even before being able to "open it to read its details.")
Therefore consider redesigning the "map note" icon,  
![X3](https://user-images.githubusercontent.com/1230959/95870092-b1911000-0d9e-11eb-98f7-fdf4763278ea.jpg)
Perhaps simply use the same icon (minus the "+") that is already used in the side panel,
![X4](https://user-images.githubusercontent.com/1230959/95870658-56135200-0d9f-11eb-88dc-905262a4b69a.jpg)
I mean let's face it, 999 out of 1000 "X"s on webpages are for button that close/destroy/remove things.
Sure, we can "remember that this one vs. the other 999 is safe, don't worry"... but it still gives us goose bumps.
(E.g., #2850 is one of the 999.)

**Comment by gravitystorm** [comment]
I also dislike the X, but not for the same reason. I find it hard to intuitively remember whether the red X means that there is an issue, or the issue has been resolved (since red X is often used to mean closed).

I propose changing the note to a yellow exclamation mark, and keeping the resolved note as a green tick.

**Comment by ScriptForAll** [comment]
Personally, I don't feel like that when I came across with an open issue note. That's simply due to the fact that when see something red with a cross my brain perceives it as "something's wrong". 
However, I do agree that using an yellow exclamation mark would make it more clear although it conflicts with the iD editor's adding a new note colour which is also yellow, as shown below:

![Screenshot_2020-10-30 OpenStreetMap](https://user-images.githubusercontent.com/36014883/97724678-206fa800-1ac5-11eb-9d7c-13f5476f73ff.png)

**Comment by GoodClover** [comment]
If we're changing the note icons I suggest also making them SVG.
The current icons are low-quality and appear badly compressed (at least for me).

**Comment by HolgerJeromin** [comment]
> And maybe a bit off topic, but it might be worth having different types of icons for different "types" of Notes. 

ref for tags on notes #5294

**Comment by kmpoppe** [comment]
This is already fixed: https://github.com/openstreetmap/openstreetmap-website/issues/2882#issuecomment-731780021

I tried designing new markers - with the existing ones as comparison:
<img width="907" height="245" alt="Image" src="https://github.com/user-attachments/assets/291520c4-5768-4d03-81d4-377ede026ed0" />

A PR should be relatively easy:
* add the SVG to app/views/layouts/_markers.html.erb
* add the colour to app/assets/stylesheets/colors.scss
* add type to app/views/layouts/map.html.erb 
* add type to app/views/notes/index.html.erb
* change icon in OSM.GetMarker in app/assets/javascripts/leaflet.map.js

which I can almost certainly do, if there are weight-ins on whether `exclam` or `question` would be the icon y'all love?

**Comment by mnalis** [comment]
> which I can almost certainly do, if there are weight-ins on whether `exclam` or `question` would be the icon y'all love?

I guess `exclam` would be better fitting, as majority of notes (which I encounter at least) ar not any questions, but instead requests for fix (missing thing, no longer existing things, roads under construction etc).

Just to make sure, that would only replace existing red `cross` icon, right?

**Comment by kmpoppe** [comment]
> Just to make sure, that would only replace existing red `cross` icon, right?

Yep, I would change the "open" marker to point to `exclam` instead of `cross`, simple switcheroo.

**Comment by kmpoppe** [comment]
While testing on the map, I felt, that yellow markers might be too light. So these are screenshots that compare the `exclam` in a red (today) and yellow (new) variant. @gravitystorm as you explicitly said "yellow exclamation mark", what are your thought on this?

<img width="367" height="227" alt="Image" src="https://github.com/user-attachments/assets/de417be5-f4fd-4c13-9d19-48f5df7c192d" />

<img width="363" height="234" alt="Image" src="https://github.com/user-attachments/assets/e45a07d7-f732-4d21-942e-c0cb2486023b" />

<img width="381" height="248" alt="Image" src="https://github.com/user-attachments/assets/ef30a3ea-d8e6-48e7-8d1e-61fee8003f32" />

<img width="423" height="283" alt="Image" src="https://github.com/user-attachments/assets/3d0e6e83-01d0-43e2-b608-1f76dc50da6b" />

Is yellow too hard to see here?

<img width="292" height="158" alt="Image" src="https://github.com/user-attachments/assets/a7bfb532-1ede-4eb2-9f67-edd992a31faa" />

**Comment by pnorman** [comment]
> While testing on the map, I felt, that yellow markers might be too light. So these are screenshots that compare the `exclam` in a red (today) and yellow (new) variant.

What are the colors you picked? Ideally in a perceptual colorspace

**Comment by kmpoppe** [comment]
> What are the colors you picked? Ideally in a perceptual colorspace

the base color (--marker-yellow in colors.scss) is `#f7d93a`, as shown in the picture above.

Additionaly, a block in the _markers.html.erb defines a "lighter" variant of the pin overlay:

```
    <linearGradient id="fill" x1="0" x2="0" y1="0" y2="40" gradientUnits="userSpaceOnUse">
      <stop offset="0" stop-color="#aaa6" />
      <stop offset="1" stop-color="#2224" />
    </linearGradient>
    <%# Lighter variant of #fill, used only by the exclam pin: a fainter wash on
        top and less darkening at the bottom, so the bright yellow note pin keeps
        its brightness/contrast instead of being muted like the default pins. %>
    <linearGradient id="fill-exclam" x1="0" x2="0" y1="0" y2="40" gradientUnits="userSpaceOnUse">
      <stop offset="0" stop-color="#fff3" />
      <stop offset="1" stop-color="#2222" />
    </linearGradient>
```

this is then picked later:

```
      <%# exclam uses the lighter #fill-exclam overlay; every other pin uses #fill. %>
      <% body_overlay = type == "exclam" ? "fill-exclam" : "fill" %>
      <g id="pin-<%= type %>" clip-path="url(#pin-clip)">
        <use href="#pin-path" fill="currentColor" />
        <use href="#pin-path" fill="url(#<%= body_overlay %>)" />
```

The thing I'm most worrying about is, that the yellow-ish pin gets buried in very busy areas.

Compare this:

<img width="688" height="450" alt="Image" src="https://github.com/user-attachments/assets/a8931b7d-7993-43eb-b864-44031f9d8b04" />

with this:

<img width="680" height="450" alt="Image" src="https://github.com/user-attachments/assets/2b4c7c06-7404-41d4-a3b2-f5fc0ea2d043" />

**Comment by mnalis** [comment]
yeah, that yellow pin color is way too invisible...  If it is to be yellow, it might need to be some much stronger / vibrant variant of it (maybe `#fffc00`?)

But that red exclamation marks looks better to me. But question is _"is red `!` less scary then red `X`?"_ for e.g. @jidanni ?

**Comment by kmpoppe** [comment]
I tried `#ffff00`, it doesn't really get better. I mean, one could always go for something minty, `#22ffdd` or so, but going away from the color-schema in use today is a big leap...

> But that red exclamation marks looks better to me. But question is _"is red `!` less scary then red `X`?"_ for e.g. [@jidanni](https://github.com/jidanni) ?

They said "I mean let's face it, 999 out of 1000 "X"s on webpages are for button that close/destroy/remove things.", so it's just the X that's scary?

**Comment by gravitystorm** [comment]
> [@gravitystorm](https://github.com/gravitystorm) as you explicitly said "yellow exclamation mark", what are your thought on this?

If the yellow doesn't work well, we can stick with the red for now. I think the exclamation mark change is the more useful part.

Trying to find an unused area of the colour space (and across 8 styles, some with dark variants) can be a challenge.

**Comment by kmpoppe** [comment]
> I think the exclamation mark change is the more useful part.

PR incoming :)

**Comment by justcivah** [comment]
It's way less scary now, thanks!

**Comment by ceirios-osm** [comment]
Personally, I find it's now more scary. Perhaps it is the colour but I agree yellow doesn't work.

**Comment by ppete2** [comment]
See also an idea to grey-out Note icons: https://github.com/openstreetmap/openstreetmap-website/issues/7383
